keda/templates/01-serviceaccount.yaml

```
@@ -11,9 +11,11 @@ metadata:
     {{- if .Values.podIdentity.azureWorkload.enabled }}
     azure.workload.identity/client-id: {{ .Values.podIdentity.azureWorkload.clientId }}
     azure.workload.identity/tenant-id: {{ .Values.podIdentity.azureWorkload.tenantId }}
-    azure.workload.identity/service-account-token-expiration: {{ .Values.podIdentity.azureWorkload.tokenExpiration }}
+    azure.workload.identity/service-account-token-expiration: "{{ .Values.podIdentity.azureWorkload.tokenExpiration }}"
```

Helm also allows you to do this:
```suggestion
    azure.workload.identity/service-account-token-expiration: {{ .Values.podIdentity.azureWorkload.tokenExpiration | quote }}
```